src/components/tile-select-group/tile-select-group.stories.ts

```diff
@@ -3,86 +3,26 @@ import { themesDarkDefault } from "../../../.storybook/utils";
-      type="${select("type", ["radio", "checkbox"], "radio", "Tile Select")}"
-      value="one"
-    >
-    </calcite-tile-select>
-    <calcite-tile-select
-      description="Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall."
-      heading="Tile title lorem ipsum"
-      icon="layers"
-      name="light"
-      ${boolean("input-enabled", true, "Tile Select") && "input-enabled"}
-      input-alignment="${select("input-alignment", ["start", "end"], "start", "Tile Select")}"
-      width="${select("width", ["full", "auto"], "auto", "Tile Select")}"
-      type="${select("type", ["radio", "checkbox"], "radio", "Tile Select")}"
-      value="two"
-    >
-    </calcite-tile-select>
-    <calcite-tile-select
-      description="Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall. Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall."
-      heading="Tile title lorem ipsum"
-      icon="layers"
-      name="light"
-      ${boolean("input-enabled", true, "Tile Select") && "input-enabled"}
-      input-alignment="${select("input-alignment", ["start", "end"], "start", "Tile Select")}"
-      width="${select("width", ["full", "auto"], "auto", "Tile Select")}"
-      type="${select("type", ["radio", "checkbox"], "radio", "Tile Select")}"
-      value="three"
-    >
-    </calcite-tile-select>
-    <calcite-tile-select
-      description="Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall.  Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall."
-      heading="TiletitleloremipsumTiletitleloremipsumTiletitleloremipsumTiletitleloremipsumTiletitleloremipsumTiletitleloremipsum"
-      icon="layers"
-      name="light"
-      ${boolean("input-enabled", true, "Tile Select") && "input-enabled"}
-      input-alignment="${select("input-alignment", ["start", "end"], "start", "Tile Select")}"
-      width="${select("width", ["full", "auto"], "auto", "Tile Select")}"
-      type="${select("type", ["radio", "checkbox"], "radio", "Tile Select")}"
-      value="four"
-    >
-    </calcite-tile-select>
-  </calcite-tile-select-group>
-`;
-
-export const Dark = (): string => html`
-<calcite-tile-select-group
-  layout="${select("layout", ["horizontal", "vertical"], "horizontal", "Tile Select Group")}"
-  dir="${select("dir", ["ltr", "rtl"], "ltr", "Tile Select Group")}"
-  class="calcite-theme-dark"
->
+const tileSelectsHTML = html`
```

Comment:
@jcfranco is this payback for calling out the others, are you trying to pin this one on me??

![nice-try](https://user-images.githubusercontent.com/10986395/191829828-0e330dd1-58ae-4869-acca-d710cc208c49.gif)

Reply:
Sorry @benelan I just saw it was merged in that PR which was yours :)

Reply:
lol all good, it was a just friendly jab at Franco because he's been having a rough time exporting stories, e.g. [here](https://github.com/Esri/calcite-components/pull/5325/commits/045e7b3136e7f40d313273082ee93837a2a0c6f5#diff-52d53222e882349bf4a846a862228cfd4f6c4764cf8c3b80d455e7532b4711e2R238), [here](https://github.com/Esri/calcite-components/pull/5325#discussion_r972392373), [here](https://github.com/Esri/calcite-components/pull/5279/commits/34c290f7b4086f376e20efaa0f67be3257993d6a#diff-646d894103b7d5b9a50d9a752cce9ac44c5a165f8f59cc541460813d8df79c2eL55)

Franco right now:
![snitch-not-absent](https://user-images.githubusercontent.com/10986395/191843772-f3a303c1-c041-4b0f-a89c-04a7a9d268e0.gif)

Reply:
Franco is now banned from touching stories :)

<img src="https://media2.giphy.com/media/ATKqmCTSd5fXygIOTB/giphy.gif?cid=790b7611d9ed9b30e1e61d576e9e0f1685d719b69560b9ea&rid=giphy.gif&ct=g" />